CHANGELOG.develop

```diff
@@ -423,6 +424,16 @@ Benner and Paweł Siudak):
 - Renamed =spacemacs/mplist-get= function to =spacemacs/mplist-get-values=,
   renamed =spacemacs/plist-get= function to =spacemacs/mplist-get-value=, and
   refactored the latter function (thanks to Benjamin Reynolds)
+- Fixed pdumper enabled on ~SPC f e R~ no matter what (thanks to Sylvain Benner)
+- Display time spent in =user-config= in home buffer (thanks to Sylvain Benner)
+- Don't call =user-load= unless we are dumping (thanks to Sylvain Benner)
```

Comment:
The `user-load` function was added as part of the dump support, so fixes related to it probably can be skipped.  Although maybe we should have an entry like "New dotfile function =dotspacemacs/user-load=", similar to the entry you added for `dotspacemacs/user-env`.